sql/src/test/java/org/apache/druid/sql/calcite/BaseCalciteQueryTest.java

```diff
@@ -190,6 +190,7 @@ public static void setupNullValues()
 
   public static final Map<String, Object> QUERY_CONTEXT_NO_STRINGIFY_ARRAY =
       DEFAULT_QUERY_CONTEXT_BUILDER.put(QueryContexts.CTX_SQL_STRINGIFY_ARRAYS, false)
+                                   .put(PlannerContext.CTX_ENABLE_UNNEST, true)
```

Comment:
Not sure how much I care about this, but I expected this to be put on the Unnest/Array tests instead of across all of the tests in this base class.

Reply:
`QUERY_CONTEXT_NO_STRINGIFY_ARRAY` is afaik only used on `CalciteArrayQueryTest` and `CalciteNestedDataQueryTest` so i think this is fine